code/modules/mob/living/living.dm

```diff
@@ -508,6 +512,7 @@
 	med_hud_set_health()
 	med_hud_set_status()
 	update_health_hud()
+	update_succumb_action()
```

Comment:
Instead of checking every single time health is updated, this could be done on demand through `set_stat()`

Reply:
Works for me, wasn't sure where to put it.